synth/src/cli/import_utils.rs

```diff
@@ -88,7 +85,15 @@ fn populate_namespace_primary_keys<T: DataSource + RelationalDataSource>(
                 table_name, primary_key.column_name
             ))?;
             let node = namespace.get_s_node_mut(&field)?;
-            *node = Content::Number(NumberContent::U64(U64::Id(Id::default())));
+            // if the primary key is a number, use an id generator.
+            let pk_node = match node {
+                Content::Number(n) => n.clone().try_transmute_to_id().ok().map(Content::Number),
+                _ => None
+            };
+            *node = Content::Unique(UniqueContent {
```

Comment:
If it has been "transmuted" into an `id`, does it still need to be wrapped in `unique`?